test/patch.jl

```diff
@@ -13,6 +13,29 @@ function strip_lineno!(expr::Expr)
    return expr
 end
 
+# Test for nested modules
+module ModA
+
+using Mocking
+
+module ModB
+
+abstract type AbstractFoo end
+
+struct Foo <: AbstractFoo
+    x::String
+end
+
+end
```

Comment:
Can we indent the submodule for clarity?

Reply:
Or even just `end # ModA` comments